tests/cli/keyboard_interface.c

```diff
@@ -148,15 +140,21 @@ int main() {
             }
         }
 
-        // Parse the buttons TODO: EXTEND TO BUTTON_XBOX
+        // Set bitmap for gamepad
         for (int i = button_a; i <= button_xbox; i++) {
-            if (buff[i] == '1') {
-                buttons |= (1 << i);
+            if (gamepad_buff[i] == '1') {
+                gamepad_buttons |= (1 << i);
+            }
+        }
+
+        // Set bitmap for keyboard
+        for(int i = 0; i < NUM_KEYBOARD_BUTTONS; i++) {
+            if(keyboard_buff[i] == '1') {
+                keyboard_buttons |= (1 << i);
```

Comment:
Can't remember if I said it, but I take back what I said a long time ago about the for loop :P
It makes sense to do
```
for (int i = 0; i < NUM_GAMEPAD_BUTTONS; i++)
```
like you did for the keyboard buttons, in case the order changes for whatever reason.